packages/components/organisms/f-content-cards/src/components/templates/ContentCardBody.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div :class="$style['c-content-card-bodyWrapper']">
+        <div :class="$style['c-content-card-body']">
+            <slot name="title">
+                <h3
+                    v-if="title"
+                    data-test-id="content-card-title"
+                    :class="$style['c-content-card-title']">
+                    {{ title }}
+                </h3>
+            </slot>
+            <slot name="subtitle">
+                <p
+                    v-if="subtitle"
+                    data-test-id="content-card-subtitle"
+                    :class="$style['c-content-card-subtitle']">
+                    {{ subtitle }}
+                </p>
+            </slot>
+            <slot name="description">
+                <template v-if="description">
+                    <p
+                        v-for="(text, index) in description"
+                        :key="index"
```

Comment:
Could this cause a problem with multiple '0' keys? Does it need to be `description-%{index}` or similar?

Reply:
not sure what your asking here, could you explain further? The index is scoped under that v-for.

Reply:
The key uniqueness problem is not a concern across components/component instances, but this could potentially be a problem if another for loop is introduced to the component at some other point. To guard against this, making the key `\`description-${index}\`` seems like a good precaution.

Reply:
> The key uniqueness problem is not a concern across components/component instances, but this could potentially be a problem if another for loop is introduced to the component at some other point. To guard against this, making the key `description-${index}` seems like a good precaution.

It would only be an issue if you were to place another loop within this loop, as its scoped to that v-for

Reply:
happy to modify if it makes people happy